pages/devices/index.js

```diff
@@ -2,8 +2,8 @@ import DeviceList from "@/components/DeviceList/deviceList";
 
 export default function deviceOverview({ devices }) {
   return (
-    <main>
+    <>
       <DeviceList devices={devices} />;
-    </main>
+    </>
```

Comment:
Do you really need a `<>` for this case? Hint: Your `<>`is surrounding just one element.